keyboards/ergotravel/keymaps/jpconstantineau/keymap.c

```diff
@@ -0,0 +1,141 @@
+﻿#include "ergotravel.h"
```

Comment:
Could you replace all of the includes here with `#include QMK_KEYBOARD_H`.

Reply:
Done